test/fixture/config/helper.conf

```diff
@@ -6,8 +6,12 @@ renew_signal = "SIGHUP"
 svid_file_name = "svid.pem"
 svid_key_file_name = "svid_key.pem"
 svid_bundle_file_name = "svid_bundle.pem"
-jwt_svid_file_name = "jwt_svid.token"
 jwt_bundle_file_name = "jwt_bundle.json"
-jwt_audience = "your-audience"
+jwt_svids = [
```

Comment:
I'm a little confused about this approach, 
the title say `allow multiple JWT audiences` but here you are not allowing that, 
you are adding a feature to create multiple JWT tokens, where each token has its own audience,
is that what you wanted? because it is possible to have multiple tokens inside the same jwt svid using [ExtraAudiences](https://github.com/spiffe/go-spiffe/blob/main/v2/svid/jwtsvid/source.go#L12)

Reply:
Yes, what I wanted was a separate token for each audience.

Reply:
We should make this clear in the readme. I created an issue to add support for the extra audiences. I think that will be useful too. https://github.com/spiffe/spiffe-helper/issues/120

Reply:
Any suggestions on documentation?

Reply:
docs updated.